### PR TITLE
Improved gcd implementation

### PR DESCRIPTION
**Pull request by piotrknapik**
Please see the example of a problematic use case and the proposed fix for it.

### CODE REVIEW

File: tests/HelpersTest.phpt [hunk 1]
```diff
@@ -5,6 +5,7 @@ use Simplex\Helpers;
 
 require_once __DIR__ . '/bootstrap.php';
 
+set_time_limit(5);
```
Comment (uestla) [inline tests/HelpersTest.phpt]: Please remove this line as it's a unnecessary side-effect in tests.

**Review by uestla — CHANGES_REQUESTED** [review]
Looks OK to me.
Could you please convert your changes to tabs as they are used in the whole project?

Thank you!

File: tests/HelpersTest.phpt (hunk 1 above)
Comment (piotrknapik) [reply tests/HelpersTest.phpt]: Time limit was added to confirm that the new test wouldn't get through without the improved code (TDD in micro scale ;) but it's totally unnecessary with the new code.

**Comment by meiokurniawan** [comment]
tell your instagram uestla, I can't run the program. I want to ask more. thank you

**Comment by graymail** [comment]
Tell your instagram uestla, I can't run the program. I want to ask more. thank you

**Pull request merged**